bridge-sdk/src/near_on_eth_client/mod.rs

```diff
@@ -0,0 +1,88 @@
+use std::sync::Arc;
+use ethereum_types::Address;
+use ethers::{contract::abigen, providers::{Http, Provider}};
+use crate::common::{Env, Error, Result};
+
+abigen!(
+    NearLightClient,
+    r#"[
+        function bridgeState() public view returns (uint,uint,uint,uint)
+        function blockHashes(uint64) public view returns (bytes32)
+    ]"#
+);
+
+pub struct NearOnEthClient {
+    eth_endpoint: String,
+    near_on_eth_client_address: Address
+}
+
+impl NearOnEthClient {
+    pub fn new(env: Env, eth_rpc_endpoint: String) -> Self {
+        match env {
+            Env::Testnet => Self {
+                eth_endpoint: eth_rpc_endpoint,
+                near_on_eth_client_address: "0x202cdf10bfa45a3d2190901373edd864f071d707".parse().unwrap()
+            },
+            Env::Mainnet => Self {
+                eth_endpoint: eth_rpc_endpoint,
+                near_on_eth_client_address: "0x3FEFc5A4B1c02f21cBc8D3613643ba0635b9a873".parse().unwrap()
```

Comment:
Move to config file

Reply:
Same as https://github.com/Near-One/rainbow-bridge-sdk-rs/pull/1#discussion_r1624115709